<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>0</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Operaci√≥n 2</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#btn_close:hover{
background-color: rgb(85, 170, 0);
}
QPushButton#btn_close{
color: rgb(0, 0, 0);
font: italic 12pt &quot;Poor Richard&quot;;
background-color: rgb(255, 255, 127);
border-radius: 5px;
}
QPushButton#btn_mostrar:hover{
background-color: rgb(85, 170, 0);
}
QPushButton#btn_mostrar{
color: rgb(0, 0, 0);
font: italic 12pt &quot;Poor Richard&quot;;
background-color: rgb(255, 255, 127);
border-radius: 5px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: italic 13pt &quot;Poor Richard&quot;;</string>
     </property>
     <property name="title">
      <string>Ingrese la fecha</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>184</width>
        <height>46</height>
       </rect>
      </property>
      <widget class="QDateEdit" name="date_fecha">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>151</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2018</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="maximumDateTime">
        <datetime>
         <hour>23</hour>
         <minute>59</minute>
         <second>59</second>
         <year>2050</year>
         <month>12</month>
         <day>31</day>
        </datetime>
       </property>
       <property name="minimumDateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2000</year>
         <month>9</month>
         <day>14</day>
        </datetime>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_mostrar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Calcular importe</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btn_close">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Cerrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
